File: backend/services/learning-service/src/main/java/com/lmspilot/learning/LearningServiceApplication.java
package com.lmspilot.learning;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class LearningServiceApplication {
    public static void main(String[] args) { SpringApplication.run(LearningServiceApplication.class, args); }
}

File: backend/services/learning-service/src/main/kotlin/com/lmspilot/learning/LearningServiceApplication.kt
package com.lmspilot.learning

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class LearningServiceApplication

fun main(args: Array<String>) {
    runApplication<LearningServiceApplication>(*args)
}
